drivers/rtc/Kconfig.stm32_rtc

```diff
@@ -0,0 +1,18 @@
+config RTC_STM32
+	bool "STM32 RTC Driver"
+	depends on SOC_FAMILY_STM32
+	default n
+	select USE_STM32_HAL_RTC
+	select USE_STM32_HAL_RTC_EX
+	select USE_STM32_HAL_RCC_EX
+	help
+	  Build STM32 RTC driver.
```

Comment:
Please add reference of the series on which it has been successfully tested

Reply:
Done